modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/RunQueryRoute.java

```diff
@@ -92,6 +95,7 @@ public class RunQueryRoute extends RunsRoute {
 
 
 	private static final GalasaGson gson = new GalasaGson();
+	private final RasDetailsQueryParams rasDetailsQueryParams;
```

Comment:
This shouldn't be a field since the query params will vary per request and this could potentially some issues when multiple requests come in at the same time